src/App.tsx

```diff
@@ -1,109 +1,32 @@
 import React from 'react';
-
+import { PostList } from './components/PostList/PostList';
```

Comment:
```suggestion
import { PostList } from './components/PostList';
```
creating index.ts in each folder would make the path shorter. It is more convenient for bigger projects.